controllers/api/user-routes.js

```diff
@@ -49,16 +49,13 @@ router.get('/:id', (req, res) => {
 
 router.post('/', (req, res) => {
   // expects {username: 'Username', email: 'example@gmail.com', password: 'password1234'}
-  User.create({
-    username: req.body.username,
-    email: req.body.email,
-    password: req.body.password,
-  })
+  User.create(req.body)
```

Comment:
Can you elaborate on why this change was made?

Reply:
because its the proper way the code should be written. Everything else is redundant

Reply:
Generally the industry standard is to explicity white list columns and fields, instead of automatically passing all the data. This is a potentional security risk if the code changes in the future and if we have a column that we don't want the user to be set or override(i.e. like an administrator flag). This also is less clear for future coders to understand. The way it was previously written was how we were taught and was industry best practice